README.rst

```diff
@@ -1,7 +1,26 @@
+flit
+====
+
+.. figure:: logo.png
+   :alt: The Flit Python packaging tool
```

Comment:
Can we have the logo on the right with text flowing around it? I think adding `:align: right` should do it.